faq/serverless-containers.mdx

```diff
@@ -143,4 +186,28 @@ solutions like Scaleway Object Storage.
 ## Why does my container have an instance running after deployment, even with min-scale 0?
 
 Currently, a new container instance will always start after each deployment, even if there is no traffic and the minimum
-scale is set to 0. This behavior is not configurable at this time.
\ No newline at end of file
+scale is set to 0. This behavior is not configurable at this time.
+
+## How can I store data in my Serverless ressource?
+
+Serverless ressources are by default [stateless](/serverless/containers/concepts/#stateless), local storage is ephemeral.
+
+In different use cases like saving results of analysis, exporting data etc... it can be important to save data. Serverless ressources can be connected to ressources from the Scaleway ecosystem.
```

Comment:
```suggestion
In different use cases like saving results of analysis, exporting data etc... it can be important to save data. Serverless resources can be connected to resources from the Scaleway ecosystem.
```